app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/item/DateSessionsSection.kt

```diff
@@ -65,6 +59,27 @@ class DateSessionsSection(private val fragment: Fragment) : Section() {
         }
     }
 
+    fun getDateHeaderPositionByDate(date: Date): Int {
```

Comment:
📝  It's better to create test code for this method. I'll create another issue to do it 👍 
It would be great you'll write the test code if you have a time. Thanks!

Reply:
OK! I’ll get it done.